masterlist.yaml

```
@@ -7385,7 +7385,7 @@ plugins:
         subs:
           - 'Keld-Nar'
           - '[Trade and Barter - Patches](https://www.nexusmods.com/skyrimspecialedition/mods/23220/)'
-        condition: 'active("Keld-Nar.esp") and not active("Trade and Barter - Keld-Nar Patch.esp")'
+        condition: 'active("Keld-Nar.esp") and not active("Keld-Nar - Trade and Barter Patch.esp")'
```

Probably best to keep both names, just for anyone that renamed the file.

```suggestion
        condition: 'active("Keld-Nar.esp") and not active("Trade and Barter - Keld-Nar Patch.esp") and not active("Keld-Nar - Trade and Barter Patch.esp")'
```

---

Oh yeah that's a good point...